score/README.md

```diff
@@ -1,5 +1,149 @@
 # Justice 40 Score application
 
+<details open="open">
+<summary>Table of Contents</summary>
+
+<!-- TOC -->
+
+- [About this application](#about-this-application)
+  - [Score comparison workflow](#score-comparison-workflow)
+    - [Workflow Diagram](#workflow-diagram)
+    - [Step 1: Run the ETL script for each data source](#step-1-run-the-etl-script-for-each-data-source)
+    - [Step 2: Calculate the Justice40 scores](#step-2-calculate-the-justice40-scores)
+    - [Step 3: Compare the Justice40 scores to other indices](#step-3-compare-the-justice40-scores-to-other-indices)
+  - [Data Sources](#data-sources)
+  - [Justice40 candidate scores](#justice40-candidate-scores)
+    - [Score A](#score-a)
+    - [Score B](#score-b)
+    - [Score C](#score-c)
+    - [Score D](#score-d)
+    - [Score E](#score-e)
+- [Running using Docker](#running-using-docker)
+- [Log visualization](#log-visualization)
+- [Local development](#local-development)
+  - [Downloading Census Block Groups GeoJSON and Generating CBG CSVs](#downloading-census-block-groups-geojson-and-generating-cbg-csvs)
+  - [Generating mbtiles](#generating-mbtiles)
+  - [Serve the map locally](#serve-the-map-locally)
+  - [Running Jupyter notebooks](#running-jupyter-notebooks)
+  - [Activating variable-enabled Markdown for Jupyter notebooks](#activating-variable-enabled-markdown-for-jupyter-notebooks)
+- [Miscellaneous](#miscellaneous)
+
+<!-- /TOC -->
+
+</details>
+
+## About this application
+
+This application is used to compare experimental versions of the Justice40 score to established environmental justice indices, such as EJSCREEN, CalEnviroScreen, and so on. The goal of this comparison is to identify the
+
+### Score comparison workflow
+
+#### Workflow Diagram
+
+TODO add mermaid diagram
+
+#### Step 1: Run the ETL script for each data source
+
+1. Call the `etl-run` command using the application manager `application.py`
```

Comment:
I'd either add a step 0 above this to tell people to set up their environment with docker or poetry as per the instructions below, or move `Score comparison workflow` to be after the setup sections.